src/anemoi/training/diagnostics/mlflow/logger.py

```diff
@@ -426,7 +432,71 @@ def _get_mlflow_run_params(
     def experiment(self) -> MLFlowLogger.experiment:
         if rank_zero_only.rank == 0:
             self.auth.authenticate()
-        return super().experiment
+
+        parent_obj = super()
+        logger_obj = self
+        experiment = super().experiment
+
+        if self._failed_to_offline:
```

Comment:
What happens if the use is already running in offline mode? That's needed in HPCs like Leonardo and MN5